src/components/nav.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+
+import NavItem from "./nav-item"
+
+const Nav = () => (
+  <nav
+    style={{
+      background: `white`,
+      marginBottom: `1.45rem`,
+    }}
+  >
+  <NavItem title="Home" link="/"/>
```

Comment:
```suggestion
    <NavItem title="Home" link="/"/>
```
Ew, indent nested items.